quickstarts/microsoft.databricks/databricks-workspace-with-customer-managed-keys-all-forms/azuredeploy.json

```diff
@@ -8,6 +8,13 @@
                 "description": "Location for all resources."
             }
         },
+        "enableNoPublicIp": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+              "description": "Specifies whether to deploy Azure Databricks workspace with Secure Cluster Connectivity (No Public IP) enabled or not"
+            }
+          },
         "workspaceName": {
             "type": "String",
```

Comment:
Can you also check if other templates have this parameter hardcoded and update them as well?

Reply:
Sure Subbu. Once I make sure the build is good for this commit, I will go ahead and update all other templates with same parameter configuration.